src/core/dbus-cgroup.c

```diff
@@ -826,6 +829,31 @@ int bus_cgroup_set_property(
 
                 return 1;
 
+        } else if (STR_IN_SET(name, "MemoryLow", "MemoryHigh", "MemoryMax")) {
```

Comment:
> Otherwise, we end up with situations where setting one knob instantiates multiple knobs or loading order issues between old and new knobs if both are defined.

Yeah, you are right.

> Can I do that in a separate PR tho?

Sure.

So, LGTM. Thanks, @htejun !